lib/gemstash/cli/start.rb

```diff
@@ -36,6 +37,12 @@ def store_daemonized
         Gemstash::Env.daemonized = daemonize?
       end
 
+      def check_rubygems_version
+        STDERR.puts("Rubygems version is too old, please update rubygems by running: " \
+                    "gem update --system") unless
+                    Gem::Requirement.new("~> 2.4").satisfied_by?(Gem::Version.new(Gem::VERSION))
```

Comment:
Yep to both, I like colors, red is so warm.

On Thu, Oct 22, 2015 at 1:25 AM, Mike Virata-Stone <notifications@github.com
> wrote:

> In lib/gemstash/cli/start.rb
> <https://github.com/bundler/gemstash/pull/32#discussion_r42699875>:
>
> > @@ -36,6 +37,12 @@ def store_daemonized
> >          Gemstash::Env.daemonized = daemonize?
> >        end
> >
> > +      def check_rubygems_version
> > +        STDERR.puts("Rubygems version is too old, please update rubygems by running: " \
> > +                    "gem update --system") unless
> > +                    Gem::Requirement.new("~> 2.4").satisfied_by?(Gem::Version.new(Gem::VERSION))
>
> Also, do you think it makes sense to colorize the message as red or yellow
> to draw more attention to it, or is that overkill?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bundler/gemstash/pull/32/files#r42699875>.
>